### PR TITLE
Notebooks/translation and editable testcases

### PR DESCRIPTION
**Pull request by chakravarthik27**
## Checklist:

- [ ] I've added Google style docstrings to my code.
- [ ] I've used `pydantic` for typing when/where necessary.
- [ ] I have linted my code
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: demo/tutorials/misc/Editing_TestCases_Notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,985 @@
...
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### Import the test cases\n",
+        "\n",
+        "harness.import_testcases() method imports the test cases from a csv file."
+      ]
+    },
+    {
```
Comment (JulesBelveze) [inline demo/tutorials/misc/Editing_TestCases_Notebook.ipynb:549]: Just add a simple markdown cell between the saving and loading back the test cases to tell the user something like: "Manually inspect your test cases and edit the test cases according to your needs"

(Simply to make it even more obvious that they need to do the changes manually)

**Review by JulesBelveze — APPROVED** [review]
Cool stuff! 🚀

File: demo/tutorials/task-specific-notebooks/Translation_Notebook.ipynb [hunk 2]
```diff
@@ -0,0 +1,1253 @@
...
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 270
+        },
+        "id": "AUZUeCpLtq-x",
+        "outputId": "c5d4327b-7ab7-4501-e4eb-1064e84ceb60"
+      },
+      "outputs": [],
+      "source": [
+        "harness.report()\n"
```
Comment (ArshaanNazir) [inline demo/tutorials/task-specific-notebooks/Translation_Notebook.ipynb:1222]: We can show harness.generated_results() here, as report is already printed in above cell.

File: demo/tutorials/task-specific-notebooks/Translation_Notebook.ipynb [hunk 3]
```diff
@@ -0,0 +1,1253 @@
...
+          },
+          "execution_count": 16,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "harness.configure(\n",
+        "    {\n",
+        "        'model_parameters': {'target_language': 'de'},\n",
```
Comment (ArshaanNazir) [inline demo/tutorials/task-specific-notebooks/Translation_Notebook.ipynb:1184]: model_parameters not needed for jsl models.

**Pull request merged**